### ISSUE TITLE
Show eXist version info

### ISSUE THREAD

**Issue by wolfgangmm** [eXist-db/dashboard#39]
Would be good to see some basic eXist version information on the dashboard main page.

**Comment by tgraham-antenna** [comment]
+1.  The only way that I know of to find the eXist version information is to use the Java client.

Putting it behind the flippable 'eXistdb' logo seems like a good place to put it.

**Comment by joewiz** [comment]
+1

@tgraham-antenna It is also currently available via the landing page of the Monitoring and Profiling app ("monex), or via any variant of the following query in eXide:

```xquery
string-join(("eXist", system:get-version(), system:get-revision(), system:get-build()), " ")
```

But Dashboard is a natural place to list this. Behind the logo or in the footer of the page.

**Comment by tgraham-antenna** [comment]
I can see it in monex now, although the only other time that I tried monex, all I got was a stack trace.  monex isn't an option if you don't happen to know a password.

It's not really a dashboard problem, but 'eXist Build' in monex is a date and the build info in 'Help > About' in the Java client is a branch and commit identifier.  Does that mean that the dashboard should provide both as in your sample query?

Running a query in eXide isn't useful for people who only want to use a provided app and who will only ever need to know the version info when they're reporting a problem to someone else.

**Comment by joewiz** [comment]
@tgraham-antenna Good point about different version information being displayed in different contexts. (Oh, and it's odd that you got a stack trace in monex. Please consider reporting it if you see this again.)

@wolfgangmm Once the revised eXist versioning practices are adopted we should do a sweep through and make sure every place that reports versions does it unambiguously.

Just one more idea along these lines. Another good location to display this would be in the eXist menu bar's System Configuration pane.

**Comment by JoernT** [comment]
fyi - i addressed this in the new dashboard. It's being displayed 'behind' an info icon on the header.

**Comment by joewiz** [comment]
"the new dashboard"?

**Comment by JoernT** [comment]
Thats available in the new dashboard on Settings page.

**Comment by adamretter** [comment]
I think it would be good if it was just present on the Dashboard from the first instance.

**Comment by joewiz** [comment]
Implemented in dashboard 2.0.
